pkg/repo/index.go

```diff
@@ -356,7 +356,7 @@ func loadIndex(data []byte, source string) (*IndexFile, error) {
 	for name, cvs := range i.Entries {
 		for idx := len(cvs) - 1; idx >= 0; idx-- {
 			if cvs[idx] == nil {
-				log.Printf("skipping loading invalid entry for chart %q from %s: empty entry", name, source)
+				slog.Info("skipping loading invalid entry for chart %q from %s: empty entry", name, source)
```

Comment:
Why was this one an info? "invalid entry" sounds like an error

Reply:
This was a 1 for 1 replacement we can bring this up to an error if it is warranted. I plan on discussing this in our developer meeting.

Reply:
fixed this.